.aegir.js

```diff
@@ -20,10 +21,19 @@ export default {
       }
     },
     before: async () => {
-      const server = createServer(undefined, {
-          ipfsModule,
-          ipfsHttpModule
-        }, {
+      /**
+       * @type {import('./src/types.js').ControllerOptions}
+       */
+      let controllerOptions = {
+        ipfsModule,
+      }
+
+      if (process.env.USE_KUBO_JS) {
+        controllerOptions.kuboRpcModule = kuboRpcModule
+      } else {
+        controllerOptions.ipfsHttpModule = ipfsHttpModule
+      }
```

Comment:
@SgtPooki  Why do we need `USE_KUBO_JS`? We don't have that for embedded JS-IPFS (`ipfsModule`).

Reply:
this is just inside .aegir.js, so it shouldn't be required by consumers. This is simply to toggle which module we're using when running tests. This PR includes changes that will use whichever option is passed in.

Reply:
See the diff, but here's links to the changes:

https://github.com/ipfs/js-ipfsd-ctl/blob/7fa9e7e8272ae4cab4e068a88ed9be8a815186fb/src/ipfsd-client.js#L97-L105
https://github.com/ipfs/js-ipfsd-ctl/blob/7fa9e7e8272ae4cab4e068a88ed9be8a815186fb/src/ipfsd-daemon.js#L103-L110
https://github.com/ipfs/js-ipfsd-ctl/blob/7fa9e7e8272ae4cab4e068a88ed9be8a815186fb/src/ipfsd-in-proc.js#L71-L80

Reply:
Eventually, we would remove USE_KUBO_JS (maybe we do so in this PR? lmk) and only run tests against kuboRpcModule, but I thought during the transition phase that we should be testing against both. open to either option